database/nakadi/10_data/04_tables/01_event_type.sql

```diff
@@ -1,7 +1,10 @@
 CREATE TABLE IF NOT EXISTS zn_data.event_type (
   et_name varchar(255) NOT NULL PRIMARY KEY CHECK (et_name <> ''),
+  et_topic varchar(255) NOT NULL,
```

Comment:
Yes... 😅 that's why I added it to the POJO. But I'm not sure if I like it. Every design problem can be solved by yet another layer, but I tried to avoid splitting event type object in a "user facing" object and in "internal object". But currently I think it's weird to have this "ignored" property that is only used internally.